test/helpers/kubectl.go

```diff
@@ -101,6 +101,9 @@ var (
 		"global.ipv6.enabled":                 "true",
 		"global.psp.enabled":                  "true",
 		"global.ci.kubeCacheMutationDetector": "true",
+		// Disable by default, so that 4.9 CI build does not panic. On 4.19 and
```

Comment:
what does `CI build does not panic` mean?

Reply:
We enable the sessionAffinity by default, and the 4.9 was failing due to LRU, so had to disable it (let me double check it).